packages/core/src/interactions/abstractInteraction.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference path="../@types/libp2p.ts" />
```

Comment:
Do we really need all these links? Is there no way to do it globally?

Reply:
And why can't we use the LibP2P types now?

Reply:
We can, but I was focussing on different tasks due to the problems with yarn and Github Actions

Reply:
Current libp2p types seem to be partially wrong and incomplete. Our types seem to fit better for the moment.